### DISCUSSION TITLE
NodeJS package-lock.json SBOM libraries list inconsistent

### DISCUSSION THREAD

**Discussion by OfirSandak** [aquasecurity/trivy#5829] [Adopters]
We need to check (and not overwrite) Dev field when retrieving uniq libs (utils.UniqueLibraries function).
See https://github.com/aquasecurity/trivy/issues/5532 for more details.

Fixed and merged 3 weeks ago:
https://github.com/aquasecurity/go-dep-parser/pull/280

Latest Trivy version doesn't include this fix, please update Trivy to use latest go-dep-parser.

**Comment by knqyf263** [comment]
You can leave a comment on https://github.com/aquasecurity/trivy/issues/5532
